airflow-core/src/airflow/api_fastapi/core_api/routes/public/xcom.py

```diff
@@ -338,3 +338,44 @@ def update_xcom_entry(
     xcom_entry.value = XComModel.serialize_value(xcom_new_value)
 
     return XComResponseNative.model_validate(xcom_entry)
+
+
+@xcom_router.delete(
+    "/{xcom_key}",
```

Comment:
https://github.com/apache/airflow/pull/58344 will probably be merged, earlier.

You'll need to do:

```suggestion
    "/{xcom_key:path}",
```

Reply:
@pierrejeambrun Looks like the other one still needs some tests to be fixed. If it gets merged before this one I will rebase and fix with your suggestion. 
So far do these current set of changes look good to you?